src/main/java/uk/ac/ebi/ega/egafuse/runner/CommandLineOptionParser.java

```diff
@@ -112,4 +107,12 @@ private static Credential readCredentialFile(Path filepath) throws IOException {
         }
         return credential;
     }
+    
+    private static Credential readCredentialConsole() throws IOException {
+        Credential credential = new Credential();
+        credential.setUsername(System.console().readLine("Enter username: "));
+        credential.setPassword(new String(System.console().readPassword("Enter password: ")));
```

Comment:
System.console().readPassword() returns "char[]". As I learned from Ashutosh, we like to store passwords as "char[]", as opposed to Strings, because a String gets interned and a "char[]" does not: https://stackoverflow.com/questions/8881291/why-is-char-preferred-over-string-for-passwords